ACadSharp/IO/DWG/DwgStreamWriters/DwgObjectWriter.Objects.cs

```diff
@@ -25,9 +25,7 @@ private void writeObject(CadObject obj)
 			switch (obj)
 			{
 				case Material:
-				case MultiLeaderStyle:
 				case MultiLeaderAnnotContext:
-				case SortEntitiesTable:
```

Comment:
Please re-add the `SortEntitiesTable` to be ignored, I still have to take a look at this one, thanks.